### ISSUE TITLE
Add Option "Orientation: vertical/horizontal" to Matrix Layout

### ISSUE THREAD

**Issue by qu4nt4r** [awawa-dev/HyperHDR#782]
#### What problem does this feature solve?

If you use HyperHDR´s Matrix-LED-Layout to build pixel displays with LED strips/strings, or you are using a common (non-square) WS2812b matrix panel for that, you are stuck due to the missing feature (i.e. you cannot run serpentines upwards/vertical). 
This makes the current software unusable for such installations.  

![2g](https://github.com/awawa-dev/HyperHDR/assets/25438135/5700b328-a659-4706-90cc-ca967f5e9fa1)

#### How should this be implemented in your opinion?

Hyperion has this option integrated:

![hyper](https://github.com/awawa-dev/HyperHDR/assets/25438135/700c608b-6be2-475e-b26c-8e5c2efae6ca)

I hope this is as simple as I think, as it adds a lot of value to the current state of software.
Cheerz,
quan

**Comment by Jerrys-modz** [comment]
This would be great. I had to use ai to flip the matrix code for mine to work

**Comment by qu4nt4r** [comment]
Yeah, thats how I "solved" it in the end, too. Nevertheless this feature
still would be of great use.

Am Do., 29. Jan. 2026 um 16:02 Uhr schrieb Jerrys-modz <
***@***.***>:
